core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -228,11 +228,16 @@ private[data] sealed abstract class KleisliInstances6 extends KleisliInstances7
     new KleisliApply[F, A] { def F: Apply[F] = A }
 }
 
-private[data] sealed abstract class KleisliInstances7 {
+private[data] sealed abstract class KleisliInstances7 extends KleisliInstances8 {
   implicit def catsDataFunctorForKleisli[F[_], A](implicit F0: Functor[F]): Functor[Kleisli[F, A, ?]] =
     new KleisliFunctor[F, A] { def F: Functor[F] = F0 }
 }
 
+private[data] sealed abstract class KleisliInstances8 {
+  implicit def kleisliDistributive[F[_], R](implicit F0: Distributive[F]): Distributive[Kleisli[F, R, ?]] =
```

Comment:
According to  naming convention of implicits, it should be `catsDataDistributiveForKleisli`